guide/improving-dev-environment/package-json-scripts.md

```diff
@@ -0,0 +1,84 @@
+# Setting up package.json scripts
+
+An easy way to run scripts like the start script, the lint script or whatever scripts you use, is by storing them in your `package.json`. After you stored these scripts in your `package.json`, you can just type `npm run start` or `npm run lint` to start your bot or lint your code!
```

Comment:
- Instead of saying "the start script" or "the lint script", I think it should be "a start script" and "a lint script", because not everyone has one or has used one in the past.
- Add "file" after "in your `package.json`" (over here and wherever else you have it)
- "After you stored" ->"After you store"